TheaterSchedule.DAL/Interfaces/IRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+
+namespace TheaterSchedule.DAL.Interfaces
+{
+    public interface IRepository<T> where T : class
+    {
+        IEnumerable<T> GetAll();
+        T Get(int id);
+        IEnumerable<T> Find(Func<T, bool> predicate);
+        void Create(T item);
+        void Update(T item);
+        void Delete(int id);
```

Comment:
Add to interface methods only needed for current PR
don't write code not related to PR